src/SprykerEco/Zed/Payone/Business/Payment/Checker/PayoneCreditCardChecker.php

```diff
@@ -72,7 +72,7 @@ public function __construct(
     public function creditCardCheck(PayoneCreditCardTransfer $creditCardData): CreditCardCheckResponseTransfer
     {
         /** @var \SprykerEco\Zed\Payone\Business\Payment\MethodMapper\CreditCardPseudo $paymentMethodMapper */
-        $paymentMethodMapper = $this->paymentMapperReader->getRegisteredPaymentMethodMapper($creditCardData->getPayment()->getPaymentMethod());
+        $paymentMethodMapper = $this->paymentMapperReader->getRegisteredPaymentMethodMapper($creditCardData->getPayment()->getPaymentMethod() ?? '');
```

Comment:
```suggestion
        $paymentMethodMapper = $this->paymentMapperReader->getRegisteredPaymentMethodMapper($creditCardData->getPaymentOrFail()->getPaymentMethodOrFail());
```
since called method was already typehinted

Reply:
mention required property at facade

Reply:
Done.